github-issue-classification/k8s/manifests/process-data.yaml

```diff
@@ -1,17 +1,17 @@
 apiVersion: "v1"
 kind: "Pod"
 metadata:
-  name: "dars-container"
+  name: ghissue-dataprocess
   namespace: kubeflow 
 spec:
   containers:
-    - name: dars-container
+    - name: dataprocess
       image: clearlinux/stacks-dars-mkl
       env:
       - name: JAVA_HOME
         value: "/usr/lib/jvm/java-1.11.0-openjdk"
       command: ["/bin/bash","-c"]
-      args: ["export JAVA_HOME=$(JAVA_HOME) && export PATH=$(JAVA_HOME)/bin:${PATH} && spark-shell -i /workdir/data/proc-data.scala"]
+      args: ["export JAVA_HOME=$(JAVA_HOME) && export PATH=$(JAVA_HOME)/bin:${PATH} && spark-shell -i /workdir/scripts/proc-data.scala"]
```

Comment:
Most of the paths are expected by the multiple scripts running in this use case. If we rename them, this might not work correctly. For now, let's not change dir names.

Reply:
@DnPlas there is no /workdir/data/ directory.